graphql/appointment/mutations.ts

```diff
@@ -161,36 +162,77 @@ export class MutateAppointmentResolver {
     @Mutation(() => Boolean)
     @AuthorizedDeferred(Role.ADMIN, Role.OWNER, Role.APPOINTMENT_PARTICIPANT)
     async appointmentDecline(@Ctx() context: GraphQLContext, @Arg('appointmentId') appointmentId: number) {
+        const { user } = context;
+        logger.info(`appointmentId: ${appointmentId}`);
         const appointment = await getLecture(appointmentId);
+        logger.info(`appointment: ${JSON.stringify(appointment)}`);
```

Comment:
Could you please add a proper log message and pass the data as object?
```suggestion
        logger.info(`appointment:`, {appointment: JSON.stringify(appointment, null, 4)});
```